PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/test/utility/AccessPrivate.java

```diff
@@ -21,12 +21,6 @@
 public final class AccessPrivate {
 
   /**
-   * Don't let anyone instantiate this class.
```

Comment:
Ok I did not know that was possible, that would indeed be a good idea!
All methods of the class are however static, so being able to instantiate the class isn't really a problem.

And to review your review: could you add  you comments to the end of the section which they apply to? So in this case to the line containing the ` }` (line 27) below. Then I can see the full code section which you comment applies to (if it isn't too big) in the pull request messages, instead of having to check the full code changes.